### PR TITLE
reproduce DoubleMuon, DoubleEG, MuonEG (partially)

### PR DESCRIPTION
**Pull request by danielhundhausen**

### CODE REVIEW

File: CrossSectionHelper.py [hunk 1]
```diff
@@ -468,7 +468,7 @@ class MCSampleValuesHelper():
             "NEvents" : NEventsValues(
                 NEVT_UL16preVFP=33861745,
                 NEVT_UL17=23075733,
-                NEVT_UL18=168618482,
+                NEVT_UL18=168594745,
```
[the remark this answers is not in the document]
Comment (danielhundhausen) [reply CrossSectionHelper.py:471]: No, this is not understood. I counted the # of events of the MINIAODs and that agrees with the DAS number. But somehow when producing this dataset some jobs always fail. Maybe this has something to do with some blocks only being available on globally blacklisted sites? This is an issue I have run into with other samples.

**Comment by danielhundhausen** [comment]
Can this be merged? We still have not managed to produce all the datasets but want to start doing MC/DATA comps via rescaling.

**Comment by finnlabe** [comment]
We should keep in mind that for this one dataset, some events are missing.

**Comment by MatthiesC** [comment]
In case this is useful: You can also pull PRs that are not merged on GitHub. In case a PR is not merged for some reason but you want to start working with it, you can create a new local branch pull the PR [1].

[1] https://stackoverflow.com/questions/27567846/how-can-i-check-out-a-github-pull-request-with-git

**Pull request merged**